lib/fs.js

```diff
@@ -1160,7 +1160,7 @@ function FSWatcher() {
   this._handle.onchange = function(status, event, filename) {
     if (status < 0) {
       self._handle.close();
-      self.emit('error', errnoException(status, 'watch'));
+      self.emit('error', errnoException(status, 'watch ' + filename));
```

Comment:
Do mean setting a filename property on the error? Looks like I could also just add an error message as the third parameter to `errnoException`